realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -48,6 +48,18 @@
 public class SyncManager {
 
     /**
+     * Debugging related options.
+     */
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+    public static class Debug {
+        /**
+         * Set this to true to bypass the device online checking.
```

Comment:
Perhaps `Set this to true to bypass checking if the device is offline before making HTTP requests`